api/src/migrations/1617825013778-CourtLocation.ts

```diff
@@ -0,0 +1,34 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CourtLocation1617825013778 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Table: court_location
+    await queryRunner.query(
+      `CREATE TABLE IF NOT EXISTS court_location (
+          id SERIAL PRIMARY KEY,
```

Comment:
Does this need co-ordinates (lat, long)?

Reply:
This is a draft only model, the details are not available right now.